packages/react/src/stateProxy.ts

```diff
@@ -44,13 +45,40 @@ export class StateProxy implements State {
         status: 'needs_identifier' as const,
         availableStrategies: [],
         isTransferable: false,
+        get firstFactorVerification() {
+          return gateProperty(target, 'firstFactorVerification', {
+            status: null,
+            error: null,
+            expireAt: null,
+            externalVerificationRedirectURL: null,
+            nonce: null,
+            attempts: null,
+            message: null,
+            strategy: null,
+            verifiedAtClient: null,
+            verifiedFromTheSameClient: () => false,
+            __internal_toSnapshot: () => {
+              throw new Error('__internal_toSnapshot called before Clerk is loaded');
+            },
+            pathRoot: '',
+            reload: () => {
+              throw new Error('__internal_toSnapshot called before Clerk is loaded');
+            },
+          });
+        },
 
         create: this.gateMethod(target, 'create'),
         password: this.gateMethod(target, 'password'),
         sso: this.gateMethod(target, 'sso'),
         finalize: this.gateMethod(target, 'finalize'),
 
         emailCode: this.wrapMethods(() => target().emailCode, ['sendCode', 'verifyCode'] as const),
+        emailLink: this.wrapStruct(
```

Comment:
Do we need `wrapStruct` because of the getter (`verification`) that was introduced?

Reply:
yeah, `emailLink` has both methods and properties. We might be able to just use `wrapStruct` in place of `wrapMethods` to be consistent though. Not sure if it's worth refactoring since this entire construct might be going away (hopefully [please])